go.mod

```diff
@@ -16,9 +16,7 @@ require (
 	github.com/getsentry/sentry-go v0.12.0
 	github.com/google/go-cmp v0.5.6
 	github.com/google/go-github/v38 v38.1.0
-	github.com/kennygrant/sanitize v1.2.4
```

Comment:
ℹ️ Now we no longer generate a package name from information in the user's `fastly.toml` manifest file, it means we don't need this dependency.